plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergStatistics.java

```diff
@@ -154,8 +154,8 @@ public void testAnalyzeWithSchemaEvolution()
                         VALUES
                           ('nationkey', null, 25, 0, null, '0', '24'),
                           ('regionkey', null, 5, 0, null, '0', '4'),
-                          ('name', 1314.0, 25, 0, null, null, null),
-                          ('info', 4417.0, null, 0, null, null, null),
+                          ('name', 1907.9999999999998, 25, 0, null, null, null),
```

Comment:
That does not look like reliably repeatable result

Reply:
Yeah, it's also different when I run the test locally.

Reply:
does it mean the test doesn't pass locally, or is flaky?

Reply:
BTW double results are asserted with rounding. you should probably have 1908.0 instead of 1907.9999999999998
(locally the test passed for me with both values; i would use the shorter one)

Reply:
> does it mean the test doesn't pass locally, or is flaky?

Consistently fails. We chatted about this test on Slack the other day

Reply:
Is it a double comparison though? I thought assertions on `SHOW STATS` are all varchars